feature/experimental/p4rt/otg_tests/traceroute_packetin_test/traceroute_packetin_test.go

```diff
@@ -343,24 +363,28 @@ func (traceroute *TraceroutePacketIO) GetPacketTemplate() *PacketIOPacket {
 	return &traceroute.PacketIOPacket
 }
 
-func (traceroute *TraceroutePacketIO) GetTrafficFlow(ate *ondatra.ATEDevice, isIpv4 bool, TTL uint8, frameSize uint32, frameRate uint64) []gosnappi.Flow {
+func (traceroute *TraceroutePacketIO) GetTrafficFlow(ate *ondatra.ATEDevice, dstMac string, isIpv4 bool, TTL uint8, frameSize uint32, frameRate uint64) []gosnappi.Flow {
```

Comment:
Is dstMac required here? I don't see it in the ATE test and the OTG API defaults to autosetting it?

Reply:
yes @jasdeep-hundal , OTG API defaults the MAC to "00:00:00:00:00:00" as per the model 

![image](https://user-images.githubusercontent.com/48308607/217233807-96963b00-0366-448c-a477-58ba6966dc97.png)<br>

__Packet capture snippet which shows the default value is set to "00:00:00:00:00:00"__
![image](https://user-images.githubusercontent.com/48308607/217233593-1a05e8cc-c993-4cb7-be57-4ef617cf047f.png)<br>

__For ATE also the destination and source MAC is added by taking the resolved Gateway MAC. But it is done by Ondatra framework implicitly__

Reply:
So the default 'choice' is 'auto', which should behave the same as the ATE API for the dest MAC. (Source MAC does need to be set specifically in OTG.)

Reply:
If we use Endpoints as Device the behavior is same as ATE, but in this test the Endpoints are port. Since traffic over port is raw traffic I have to feed in the destination MAC address.

I can't add Device as TxRx for this test as Src and Dst are on the same port. The server will throw an error as below. 
```- Tx device name "atePort1.IPv4" is same as Rx device name in flow "IP4"```

![image](https://user-images.githubusercontent.com/48308607/217601420-d972402c-2b12-4cb0-b22e-f89c2c6ddec1.png)